### ISSUE TITLE
No email notification on failing build

### ISSUE THREAD

**Issue by ChristianKuehnel** [google/llvm-premerge-checks#317]
**Describe the bug**
The pre-merge build of my Revisions failed with Phab showing the error. However I did not get an email notification. So I actively have to check the UI to see the failing builds.

Relevant links: 
* https://reviews.llvm.org/D105014 (some of the patches had problems with applying them)
* https://reviews.llvm.org/D105177 (Windows build is failing)

**To Reproduce**
1. Submit a patch with failing builds
2. Wait for an email notification from Phabricator

**Expected behavior**
On every failing build of the pre-merge tests I would like to receive an email telling me that it failed and where I can find the build results.

I could imagine that some folks do not want to be bothered with notifications around failing builds, however I would like to know ASAP so that I can fix these. 

**Screenshots**

**Comment by metaflow** [comment]
I agree that it should be useful in general to get build results if build failed (but not e.g. cancelled as a new revision was uploaded).
We can implement it by opt-in or opt-out basis. For the beginning I suggest to use existing group https://reviews.llvm.org/project/view/78/ membership as a indication that one want's to particiate.
Bot will add a comment to a revision like we did some time ago (but disabled as some people were complaining).
What do you think?

**Comment by ChristianKuehnel** [comment]
Sounds like a reasonable plan. I just surprised that there is no out of the box way for build failure notification in Phabricator.

@joker-eph are you aware of any notification mechanism we could use?

**Comment by joker-eph** [comment]
I haven't studied this in Phab.
